src/DocRaptor.php

```diff
@@ -71,10 +67,19 @@ public function setDocumentUrl($document_url)
     /**
      * @param string $document_type
      * @return $this
+     * @throws Exception
      */
     public function setDocumentType($document_type)
     {
+        $allowedValues = array('xls', 'xlsx', 'pdf');
```

Comment:
Array constants exist only from PHP 5.6 but we can define single constants for the formats. Thoughts?